bin/check-graylog-buffers.rb

```diff
@@ -37,6 +37,12 @@
 require 'rest-client'
 
 class CheckGraylogBuffers < Sensu::Plugin::Check::CLI
+  option :protocol,
+         description: 'Protocol for connecting to Graylog',
+         short: '-proto',
+         long: '--protocol PROTOCOL',
+         default: 'http'
```

Comment:
As much as I feel `https` should be the default I agree that the breaking change would be unwarranted right now.